components/navigation/navItem.js

```diff
@@ -1,89 +1,87 @@
 import React from "react";
-
-import Link from "next/link";
 import { urlInChildren } from "../../lib/utils.cjs";
-
 import NavChild from "./navChild";
 
-export default class NavItem extends React.Component {
-  constructor(props) {
-    super(props);
-  }
-
-  trueName() {
-    return this.props.page.path.split("/").pop();
-  }
-  cleanName() {
-    return this.trueName().replaceAll("-", " ");
-  }
-  shouldHaveID() {
-    return this.offScreen ? this.page.name : false;
-  }
+const NavItem = ({
+  page,
+  offScreen,
+  slug,
+  condensed,
+  paths,
+  version,
+  maxVersion,
+}) => {
+  const trueName = () => {
+    return page.path.split("/").pop();
+  };
 
-  render() {
-    const props = this.props;
+  const cleanName = () => {
+    return trueName().replaceAll("-", " ");
+  };
 
-    let subNav;
-    let url_in_child = false;
+  const shouldHaveID = () => {
+    return offScreen ? page.name : false;
+  };
```

Comment:
`shouldHaveId` could just be a constant, no?

```
const shouldHaveId = offScreen ? page.name : false;
```

Reply:
In fact, these weren't being used, so I deleted them on 51f922b